Makefile

```diff
@@ -62,7 +62,7 @@ push:	docker
 
 # Run unit tests and skip the BDD tests
 unit-test: golint check
-	( cd service; go clean -cache; CGO_ENABLED=0 GO111MODULE=on go test -skip TestGoDog -v -coverprofile=c.out ./... )
```

Comment:
Hi @sakshi-garg1 Removing this `-skip TestGoDog` will result in running BDD tests as well as a part of unit-tests and it may fail